### PR TITLE
Add feature flag memberships to user summary information

### PR DESCRIPTION
**Pull request by karlbaker02**
This PR adds feature flag memberships to the user summary information, both for user-specific memberships and for those which apply to all users.

This update is made to allow for changes in services which use feature flags to control logic and specifically, to remove the need to pass feature flag information between services (such as between MAAT and the Orchestration Service). The feature flag information is added specifically to the `UserSummaryDTO` as this is the most appropriate place to hold this information, similar to that held already for role data items.

[Part of LCAM-1342](https://dsdmoj.atlassian.net/browse/LCAM-1342)

### CODE REVIEW

File: maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "FEATURE_TOGGLE", schema = "TOGDATA")
+public class FeatureToggleEntity {
+
+  @Id
+  @Column(name = "ID", nullable = false)
```
Comment (venkat980) [inline maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java:21]: Looking at the FEATURE_TOGGLE table, ID uses a sequence. Should we keep the entity inline with the DB table please?

**Review by venkat980 — COMMENTED** [review]
Looks good to me, just one comment.

File: maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java (hunk 1 above)
Comment (karlbaker02) [reply maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java:26]: Done. I'm not sure if the sequence name here needs to match one that already exists or not - I couldn't see anything in the db. If you could confirm that'd be great, thanks.

Comment (venkat980) [reply maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FeatureToggleEntity.java:26]: Looking at the DB, the sequence we are using is ISEQ$$_316195

**Review by venkat980 — APPROVED** [review]

**Pull request merged**